MobileWallet/Common/NotificationManager.swift

```diff
@@ -129,7 +129,7 @@ class NotificationManager {
     func handleForegroundNotification(_ notification: UNNotification, completionHandler: (UNNotificationPresentationOptions) -> Void) {
         ConnectionMonitor.shared.state.baseNodeSynced = nil
         try? TariLib.shared.tariWallet?.syncBaseNode()
-        //completionHandler([.alert, .badge, .sound])
+        completionHandler([.alert, .badge, .sound])
```

Comment:
I think this'll start showing all users notifications in the apps foreground

Reply:
yes, you're right. Thx